corehq/apps/builds/README.md

```diff
@@ -1,4 +1,40 @@
-Adding CommCare Builds to CommCare HQ
+Syncing local HQ instance with an Android Phone
```

Comment:
Moved android instructions to the top, since nobody uses (or should use) J2ME anymore